### ISSUE TITLE
Unable to set executable path from WSL 2 directory

### ISSUE THREAD

**Issue by Wwarrior1** [lensvol/intellij-blackconnect#41]
**Problem: Lack of support for WSL 2**
Unable to neither set Local Instance's path `\\wsl$\Ubuntu-20.04\home\...\venv\bin\blackd` nor run instance via Start button.
Trying to save settings fails with an error `'blackd' binary path does not point to an executable.`

![image](https://user-images.githubusercontent.com/12012058/183418562-5e0fe3c1-ce51-4b79-b147-e5bfd065fcb7.png)

It would be nice to support WSL 2 🙏

**Comment by lensvol** [comment]
WSL 2 was never in the plans and I do not have an easy way to test it, sadly. Can you please share your exact OS information (Windows, Linux distro, black version, its method of install?).

**Comment by Wwarrior1** [comment]
Right, so I can run instance manually then. Hopefully plugin works with it.
Setup: venv created inside WSL/Ubuntu via `python3 -m venv venv`. Black(d) 22.3.0. Windows 10.

**Comment by gilsonbp** [comment]
I have the same problem.
I didn't want to have to install python on windows just to run Black.
I've tried to do this in a few ways like giving the WSL executable path or passing it from `.venv` but none of those ways seem to work.
@lensvol did you manage somehow?

**Comment by j7zAhU** [comment]
I'm seeing the same problem. Would be nice to find a way to use it

**Comment by DaniinXorchenabo** [comment]
+
I get this problem too.
I installed this plugin to `PyCharm 2022.1.4` (Windows 10) and I don't used wsl  (but wsl 2 running in my computer)
